.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "livePreview.defaultPreviewPath": "/src/index.html"
+}
```

Comment:
- [ ] Please remove the folder as suggested by the previous reviewer to make your repo clean.